.eslintignore

```diff
@@ -1,3 +1,4 @@
 build/
 node_modules/
-src/types
\ No newline at end of file
+src/types
+generated/
```

Comment:
We didn't include generated since the scripts we use in this repo make sure to send the generated files to `src/types`.
It could be included without any issues, just a quick headsup. (also, it's usually a good idea to use those scripts, since this subgraph uses manifest template, and requires mustache to process a few files before getting usable files for deployment).

We could probably improve the dev workflow though.

Reply:
Well, I'm using scripts in the package.json and I have a `generated` directory. 🤷 
Worth figuring out what I'm doing differently there, but for now it's not hurting anything to make this update to .eslintignore.